synapse/storage/databases/main/schema/delta/59/drop_device_max_stream_id.sql

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C
```

Comment:
firstly, this file is in the wrong directory: the current schema version (as per `prepare_database.py`) is 58. (I see we have a file that has crept into the wrong directory by mistake: I'll follow up on that elsewhere.)

Secondly, delta files should be numbered to make sure they run in the right order. Please could you rename it to `21drop_device_max_stream_id.sql` to fit in https://github.com/matrix-org/synapse/tree/develop/synapse/storage/databases/main/schema/delta/58?

Thirdly, as you don't work for the matrix.org foundation, you can't write this copyright line. You can either put your own name there or omit the copyright line altogether.

Reply:
Thanks for the notes @richvdh !

File moved to directory (I found the folder and assumed that {version+1} would be the migrations for the next version so I originally put it there.

Regarding the copyright, I thought that with the notice I would be "surrendering" the copyright to the Matrix.org foundation; apparently that was not the case.

File renamed & moved under version 58, and copyright notice removed, as requested.